app/controllers/items_controller.rb

```diff
@@ -210,22 +209,6 @@ def catkey
     end
   end
 
-  def tags_bulk
```

Comment:
Now that this is no longer a concern in the items controller, I wonder if lines 446-449 can change to be:

```ruby
return if params[:bulk]
```